### PR TITLE
[PLAY-23]: configure CI/CD for the repo

### PR DESCRIPTION
**Pull request by AntonNos**

### CODE REVIEW

**Review by Christopher-Gustavsson — APPROVED** [review]
The changes are looking good to me. I don't see anything that looks out of place here.
I'd wait for Daniels approval as well before merging as he has more experience with github actions.

**Comment by AntonNos** [comment]
> The changes are looking good to me. I don't see anything that looks out of place here. I'd wait for Daniels approval as well before merging as he has more experience with github actions.

Thank you for comment! Yes, lets wait Daniels approval

File: jestconfig.json
Comment (dechegaray) [inline jestconfig.json:21]: What is this tool for? just curiosity

**Review by dechegaray — APPROVED** [review]
Looks good!

File: jestconfig.json
Comment (AntonNos) [reply jestconfig.json:21]: It is a plugin for jest, that allows generating XML reports. I read that this may be helpful for CI/CD tools

Comment (AntonNos) [reply jestconfig.json:21]: BTW, if we don't need it, I can remove it.

**Pull request merged**
